scripts/irods/test/test_iadmin.py

```diff
@@ -1708,6 +1708,137 @@ def test_iadmin_exits_nonzero_on_nonexistent_command__issue_7734(self):
         ec, _, _ = self.admin.assert_icommand(['iadmin', 'nonexistentcommand'], 'STDERR');
         self.assertNotEqual(ec, 0)
 
+    def test_iadmin_rmuser_removes_associated_acls__issue_7778(self):
+        test_user_name = "test_user_issue_7778"
+        test_group_name = "test_group_issue_7778"
+        test_collection_name = "test_collection_issue_7778"
+        test_dataobj_name = "test_data_object_issue_7778"
+        try:
+            self.admin.assert_icommand(["imkdir", test_collection_name])
+            self.admin.assert_icommand(["itouch", test_dataobj_name])
```

Comment:
Is this data object cleaned up because it's in the session collection? Just making sure since I noticed we are running `irmdir` on the test collection but not removing the test data object.